aixplain/modules/pipeline/designer/enums.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+
+
+class DataType(str, Enum):
```

Comment:
There is already a data type enumerator on `aixplain/enums`. You may update (with the missing values) and use that one to avoid redundancy.

Reply:
Done